hw/ip/i2c/doc/_index.md

```diff
@@ -23,9 +23,7 @@ See that document for integration overview within the broader top level system.
     - 7-bit target address
 - Support for the following optional capabilities:
     - Clock stretching in the host mode
-    - Automatic clock stretching in the target mode when TX FIFO is empty during a read<sup>2</sup>
-    - Automatic clock stretching in the target mode when ACQ FIFO is full during a write
-    - Clock stretching in the target mode after a complete address acquire, byte read, or byte write transactions
+    - Automatic clock stretching in the target mode<sup>2</sup>
```

Comment:
Nit: should this bullet point end in a period?

Reply:
ah for this one i just followed the previous bullets, didn't want to make this one stand out.